lib/core/keys.dart

```diff
@@ -90,6 +112,31 @@ String printKeyEvent(KeyboardEvent event) {
   return buf.toString();
 }
 
+String makeKeyPresentable(String key) {
+  List keyAsList = key.split("-");
+  if (isMac()) {
+    if (keyAsList.any((s) => s == "meta")) {
+      return null;
+    }
+    keyAsList = keyAsList.map((s) {
+      if (_unicodeMac.containsKey(s)) {
+        return _unicodeMac[s];
+      } else {
+        return capitalize(s);
+      }
+    }).toList();
+    return keyAsList.join("&thinsp;");
+  } else {
+    if (keyAsList.any((s) => s == "macctrl")) {
+      return null;
+    }
+    keyAsList = keyAsList.map((String s) => capitalize(s)).toList();
+    return keyAsList.join("+");
+  }
+}
+
+String capitalize(String s) => '${s.substring(0,1).toUpperCase()}${s.substring(1)}';
```

Comment:
This is defined twice. Can you move it to somewhere like `src/util.dart`? Also we should cover the cases of empty strings and strings of length 1.